baseline/OSPS-SA.yaml

```diff
@@ -131,6 +137,13 @@ controls:
           - 6.2.1
           - 12.3.1
           - 12.8.1
+      - reference-id: 800-161
+        identifiers:
+          - CM-2
+          - PL-2
+          - PL-8
+          - RA-3
+          - SA-15           
```

Comment:
As I read it, this OSPS-SA-02 is about the interfaces of the released software, whereas NIST's SA-15 is more about the process for building it.

Reply:
Having the data from SA-01 & SA-02 help downstream consumers make choices in selecting and implementing software.  Their review of this documentation is part of THEIR build process.  How the project operates is something downstream should understand and plan accordingly for if upstream practices do not align with their downstream requirements (e.g. regulations on the Manufacturer)